flow/src/main/java/flow/Flow.java

```diff
@@ -43,7 +43,11 @@
   }
 
   @NonNull public static Flow get(@NonNull Context context) {
-    return InternalContextWrapper.getFlow(context);
+    Flow flow = InternalContextWrapper.getFlow(context);
+    if (null == flow) {
+      throw new IllegalStateException("The passed context must be wrapped with Flow!");
```

Comment:
It would be helpful if this message carried an explanation of what needed to be done, but wasn't: Setting up Flow in the Activity, with a rough description of what's missing.